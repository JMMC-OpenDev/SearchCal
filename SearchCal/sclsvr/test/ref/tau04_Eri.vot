version https://git-lfs.github.com/spec/v1
oid sha256:e5dc71742adecce08b15eef915395f28fbd76c1d1d0c2a32fbae8214f5069ec2
size 145438
