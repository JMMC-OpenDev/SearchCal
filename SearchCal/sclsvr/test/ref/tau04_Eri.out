version https://git-lfs.github.com/spec/v1
oid sha256:bf00c5549522f324b96cda636e692a16667070f2aba06544cc9126cea816d6c8
size 218533
